### PR TITLE
Nicolas/avails add to selection

### PR DESCRIPTION
**Pull request by n-germain-work**
State issue number in your PR to close it when PR merged: issue #XXX
e.g. : "Resolve the bundle size ❤️: issue #1234"

### CODE REVIEW

File: apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import {
   selectDates,
   selectNonExclusive,
   selectExclusive,
+  assertInputValue,
```
Comment (phpgeekfr) [inline apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts:21]: unused import

**Review by phpgeekfr — APPROVED** [review]

File: apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts (hunk 1 above)
Comment (n-germain-work) [reply apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts:21]: it is line 89

Comment (phpgeekfr) [reply apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts:21]: no !!!!!!

**Review by phpgeekfr — APPROVED** [review]

File: apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts (hunk 1 above)
Comment (n-germain-work) [reply apps/catalog/catalog-e2e/src/integration/dashboard/avails-search.spec.ts:21]: haha my bad ^^

**Pull request merged**